app/lib/konnectors.js

```diff
@@ -0,0 +1,61 @@
+/* konnector lib ready to be added to cozy-client-js */
+const KONNECTORS_DOCTYPE = 'io.cozy.konnectors'
+
+const STATE_READY = 'ready'
+
+export function addAccount (cozy, konnector, account) {
+  return Promise.resolve(konnector)
+}
+
+export function fetchManifest (cozy, source) {
+  return cozy.fetchJSON('GET', `/konnectors/manifests?Source=${encodeURIComponent(source)}`)
```

Comment:
`cozy` means the global `cozy` here? why not `cozy.client.fetchJSON()`? It seems to be a method specific for client here.